scripts/ci/debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
This file is fine. The source format file specifies whether dpkg is expecting a separate "source" tarball and "packaging" metadata, or whether both are expected in the same tarball. `native` is correct for the latter, which is what we're doing.